File: Introduccion a Poo/Ejercicio 1/Java/Persona.java
public class Persona {
    private String nombre;
    private int edad;
    private String ciudad;
    public Persona(String nombre, int edad, String ciudad) {
        this.nombre = nombre;
        this.edad = edad;
        this.ciudad = ciudad;
    }
    public void saludo() {
        System.out.println("Hola, soy " + nombre + " de " + ciudad);
    }
    public boolean esMayorDeEdad() {
        return edad >= 18;
    }
}

File: Introduccion a Poo/Ejercicio 1/Python/Persona.py
class Persona:
    def __init__(self, nombre, edad, ciudad):
        self.nombre = nombre
        self.edad = edad
        self.ciudad = ciudad
    def saludo(self):
        print("Hola soy", self.nombre, "de", self.ciudad)
    def mayor(self):
        if self.edad >= 18:
            return True
        else:
            return False
persona1 = Persona("MArce", 25, "Inglaterra")
persona2 = Persona("Jesus", 17, "Portugal")
persona3 = Persona("Villalobos", 30, "Madrid")
persona1.saludo()
persona2.saludo()
persona3.saludo()
print(persona1.mayor())
print(persona2.mayor())
print(persona3.mayor())
